gcloud/bigquery/client.py

```diff
@@ -96,3 +97,20 @@ def dataset(self, name):
         :returns: a new ``Dataset`` instance
         """
         return Dataset(name, client=self)
+
+    def load_from_storage(self, name, destination, *source_uris):
```

Comment:
7f38c28751ac92c42fe450e013f032832219f98a renames the class and the corresponding client factory to include "table".